src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+// eslint-disable-next-line promise/param-names
+const promise2 = new Promise((_, reject) => {
```

Comment:
The underscore `_` is used as a placeholder for the unused `resolve` parameter, which is fine. However, ensure that this is intentional and aligns with your code style guide.